### PR TITLE
Look fixes and Ship button deactivating

### PR DESCRIPTION
**Pull request by horinezachary**
If you disagree with any of these, feel free to change them, I was looking at stylistic choices that we could make, and trying to keep our formatting uniform between all of our elements.

### CODE REVIEW

File: src/main/java/assets/css/custom.css [hunk 1]
```diff
@@ -211,6 +211,9 @@ td:hover {
     margin: 0px;
     border-width: 0px;
 }
+.ship-placed {
+    opacity: 0.5;
```
Comment (johnnybecerra) [inline src/main/java/assets/css/custom.css]: I like this change. It looks significantly better this way

**Comment by horinezachary** [comment]
I am going to wait to merge this until Michael takes a look at it, because I did make some visual changes to his elements.

File: src/main/java/assets/game.js [hunk 2]
```diff
@@ -209,16 +209,22 @@ function initGame() {
...
     document.getElementById("place_destroyer").addEventListener("click", function(e) {
+    document.getElementById("place_destroyer").disabled = true;
+    document.getElementById("place_destroyer").classList.add("ship-placed");
         shipType = "DESTROYER";
        registerCellListener(place(3));
        placingMode = 2;
     });
     document.getElementById("place_battleship").addEventListener("click", function(e) {
+    document.getElementById("place_battleship").disabled = true;
+    document.getElementById("place_battleship").classList.add("ship-placed");
```
Comment (montymxb) [inline src/main/java/assets/game.js]: This is potentially bugged, as are the other 2 spots above this. If you click a ship, and click another ship without placing the 1st you won't be able to go back and place it. The result is that the game is stuck,  and the page has to be reloaded to continue.

Comment (horinezachary) [reply src/main/java/assets/game.js]: I didn't think about that, thanks!

Comment (montymxb) [reply src/main/java/assets/game.js]: @horinezachary I would check out adding this change to the `cellClick` function, around line 112 (you should see the battleReport addition there). If that fires then we know we have a given ship that's been placed, so you can add a few ifs to handle the case for disabling each ship as it's placed.

**Comment by fireball1983** [comment]
I'll be in work/class all day, so make changes to style freely.

**Comment by montymxb** [comment]
I've taken the liberty of fixing the bug mentioned before. Going to merge this so we can get #70 on the way. Outstanding UI issues can be corrected after that.

**Comment by horinezachary** [comment]
oops, went to push, and it said I was behind master because you already did it

**Comment by montymxb** [comment]
@horinezachary you can pull down to sync up changes before you push again.

**Comment by horinezachary** [comment]
I pulled the fix into a separate function to increase code readability

File: src/main/java/assets/game.js [hunk 3]
```diff
@@ -138,6 +146,16 @@ function cellClick() {
     }
 }
 
+function setDisabled(shipType) {
+    if (shipType === "MINESWEEPER"){shipId = "place_minesweeper";}
+    else if (shipType === "DESTROYER"){shipId = "place_destroyer";}
+    else if (shipType === "BATTLESHIP"){shipId = "place_battleship";}
+    else {console.warn("!! Unknown ship found in lockout code, '"+shipType+"' !!"); return;} // warn of an unknown ship type
```
Comment (horinezachary) [inline src/main/java/assets/game.js]: While it might never occur, this was a good call.

Comment (horinezachary) [reply src/main/java/assets/game.js]: (the error check)

**Comment by montymxb** [comment]
Added that just in case we add another ship type, and forget to change this
code. Also if the code is good feel free to merge it in. I'm busy until
about 2:20 or so, at which point I'll wrap everything up.

**Pull request merged**